application-tour-ui/src/main/resources/TourCode/StepSheet.xml

```diff
@@ -563,7 +563,10 @@ if(typeof(XWiki) == 'undefined' || typeof(XWiki.widgets) == 'undefined' || typeo
    display: block;
    padding: .3em .3em .3em 20px;
    float:right;
-}</code>
+}
+.tour-backdrop {
+   opacity: 0.5 ;
```

Comment:
Hi,
There shouldn't be a space between `0.5` and `;`.
And while you're at it, you can also add a space after the `float:` above so the code style is consistent.

Reply:
Hi! Thanks for your suggestions. Let me make these changes.

Reply:
@9inpachi Now please have a look at the changes. Thank you!